src/main/resources/db/migration/V1_12__create_external_object_document_directory_table.sql

```diff
@@ -0,0 +1,18 @@
+CREATE TABLE IF NOT EXISTS external_object_directory (
+  eod_id                         INTEGER                     NOT NULL
+, moj_med_id                     INTEGER
+, moj_tra_id                     INTEGER
+, moj_ann_id                     INTEGER
+, external_location              CHARACTER VARYING           NOT NULL
+, external_location_type         CHARACTER VARYING           NOT NULL
+, created_ts                     TIMESTAMP WITH TIME ZONE    NOT NULL
+, modified_ts                    TIMESTAMP WITH TIME ZONE    NOT NULL
+, modified_by                    INTEGER                     NOT NULL
+, status                         INTEGER                     NOT NULL
```

Comment:
Please check with @hemantasharma1129  - whether he expects us to be adding a "xxx_status_type" table as well?